lib/src/scanner_type.dart

```diff
@@ -0,0 +1,53 @@
+enum ScannerType {
+  unknown,
+  bluebird,
+  fastBarcode,
```

Comment:
Maybe we should use camera here, since fastBarcode is not clear for someone who doesn't have experience with it 🤔